dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -97,17 +105,20 @@ public boolean equals(Object obj) {
             return false;
         }
         final WorkflowVersion other = (WorkflowVersion)obj;
-        return Objects.equals(super.getName(), other.getName()) && Objects.equals(super.getReference(), other.getReference());
+        return Objects.equals(super.getName(), other.getName()) && Objects.equals(super.getReference(), other.getReference())
+                && Objects.equals(this.lastModified, other.lastModified);
     }
 
     @Override
     public int compareTo(WorkflowVersion that) {
-        return ComparisonChain.start().compare(this.getName(), that.getName(), Ordering.natural().nullsLast()).compare(this.getReference(), that.getReference(), Ordering.natural().nullsLast()).result();
+        return ComparisonChain.start().compare(this.getName(), that.getName(), Ordering.natural().nullsLast())
```

Comment:
Same as previous comment. Also, I think you could call `super.compareTo` to avoid duplication.

Reply:
got rid of methods in version instead